src/tests/D3MHub.t.sol

```diff
@@ -770,6 +896,144 @@ contract D3MHubTest is DSSTest {
         assertEq(vat.sin(vow), sinBefore + 40 * RAD);
     }
 
+    function test_wind_pool_caged() public {
```

Comment:
Mmm what is the difference between this and the existing `test_unwind_pool_caged`?

So there is already a `test_unwind_*` series of tests, named after the fact that the action we do (not explicilty unwinding) will cause an unwind.

The new `test_wind_*`  series is basically the same, and confused me. We should try to merge them to the existing format.

Reply:
Yeah these two seems to be equivalent. I guess the naming is subjective, but I prefer the naming that reflects what actually happens than the what would you might expect to happen but are testing is not the case.

Reply:
So in this case, pool is caged, so what you are testing is that does NOT wind more and actually unwinds.